context/myuuid.go

```diff
@@ -11,13 +11,14 @@ import (
 )
 
 var (
-	myUUIDFile = path.Join(config.GetString("localstatedir"), "uuid")
+	myUUIDFile string
 )
 
 // InitMyUUID initializes MyUUID by reading the `<config.LocalStateDir>/uuid` file.
 // If the file is not present it generates a new UUID and saves it to the file.
 // If the file is not accessible. initMyUUID panics.
 func InitMyUUID() uuid.UUID {
+	myUUIDFile = path.Join(config.GetString("localstatedir"), "uuid")
```

Comment:
Correct. Other changes are cosmetic changes: One is to rename method 'dumpConfig', other is usage of `SetDefault` which takes effect only if no command line arg or conf is provided and no default is provided. But we can provide default value during `parseFlags` itself which only called during runtime.